docker/quickstart/quickstart_version_mapping.yaml

```diff
@@ -0,0 +1,18 @@
+# quickstart versioned by git sha/tag
+# docker is version by release tag and head 
+# 
+# if --version is specified during CLI run 
+# the CLI will try to use the same quickstart and docker tags
+# quickstart_mapping maps broken releases to fixed ones
+quickstart_version_map:
+  # default key is mandatory and is used if no version is specified
+  default:
+    composefile_git_ref: master
+    docker_tag: head
+  # v0.9.6 images contain security vulnerabilities 
+  v0.9.6: 
+    composefile_git_ref: v0.9.6.1
+    docker_tag: v0.9.6.1
+  stable: 
+    composefile_git_ref: master
+    docker_tag: head
```

Comment:
should stable be pointing at head/master?

Reply:
in an ideal world yes. changed this, if not defined fetches the latest release tag from github